### DISCUSSION TITLE
OTEL agent receiver authenticator

### DISCUSSION THREAD

**Discussion by panthamm** [open-telemetry/opentelemetry-collector#4394] [Q&A] [answered]
OTEL agent receivers side authenticator is only working with GRPC (not with HTTP). Is it expected?

**Comment by panthamm** [comment, marked as answer]
Received response from Slack OTEL collector channel. Currently, it is supported GRPC.

**Reply by pavankrish123** [reply]
The support for HTTP is added in PR #4506
